recipes/cell2cell/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "cell2cell" %}
+{% set version = "0.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cell2cell-{{ version }}.tar.gz
+  sha256: 2e17a1556d90c187434aa492000b2ced0d755cf6d437732f07df42e6670370e0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("cell2cell", max_pin="x.x.x") }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+
+  host:
+    - python ==3.10.*
+    - cython
+    - pip
+  run:
+    - python ==3.10.*
+    - numpy >=1.16,<2.0
+    - pandas >=1.0.0
+    - xlrd >=1.1
+    - openpyxl >=2.6.2
+    - networkx >=2.3
+    - matplotlib-base >=3.2.0,<3.7.3
+    - seaborn >=0.13.0
```

This conflicts with https://github.com/conda-forge/statannotations-feedstock/blob/main/recipe/meta.yaml#L24

```suggestion
    - seaborn
```